### PR TITLE
Fixed typo in sentence

### PR DESCRIPTION
**Pull request by sabahshariq**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/core/tools/csproj.md [hunk 1]
```diff
@@ -263,7 +263,7 @@ Relative or absolute path to the *.nuspec* file being used for packing.
 Base path for the *.nuspec* file.
 
 ### NuspecProperties
-Semicolon separated list of key=value pairs.
+Semicolon separated list of key-value pairs.
```
Comment (mairaw) [inline docs/core/tools/csproj.md]: the key=value pairs is correct. This is the format used for this property.

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @ThePiranha. The removal of the double which looks good but the second file was correct actually. Can you undo that change please?

**Comment by ThePiranha** [comment]
@mairaw cleaned up the other commits. Now one commit for one file change.

**Review by mairaw — APPROVED** [review]
Didn't need to clean the commit but thank you! 💯

**Pull request merged**